ref-tests/src/utils.rs

```diff
@@ -105,21 +105,20 @@ pub async fn create_wallet_canister(agent: &Agent) -> Result<Principal, Box<dyn
 
     #[derive(CandidType)]
     struct Input {
-        num_cycles: candid::Nat,
-        num_icpt: candid::Nat,
+        num_cycles: Option<candid::Nat>,
```

Comment:
@akhi3030 I think you need to name this `amount` instead of `num_cycles`. `provisional_create_canister_with_cycles` expects an `amount` field based on its Candid interface:
```
provisional_create_canister_with_cycles :
    (record { amount: opt nat }) -> (record {canister_id : canister_id});
```
I'm not sure how the tests work for you locally -- I'd say they shouldn't.

Reply:
Good catch @dsarlis.  Now updated.

Reply:
Well, there are other failures now but at least this one was fixed.

Reply:
Yeah, this failure is related to `cdk-rs` repo I believe.

Reply:
Hmm, ok, from the looks of it this PR has to wait until both `cdk-rs` and `wallet-rs` are updated to use the new system api. Some tests here are using the wallet wasm and it's still importing the old api.